makefile.js

```diff
@@ -12,18 +12,16 @@ const load = value => {
 const darwin = load('darwin');
 const win32 = load('win32');
 
+const data = Object.entries(darwin).map(([name, figure]) => [name, figure, win32[name]]);
+
+// TODO: Use spread syntax when targeting Node 8.3
 const jsonTable = [
 	[
 		'Name',
 		'Real OSes',
 		'Windows'
 	]
-];
-
-// TODO: Use `Object.entries` when targeting Node.js 8
-for (const key of Object.keys(darwin)) {
-	jsonTable.push([key, darwin[key], win32[key]]);
-}
+].concat(data);
```

Comment:
You could use array spreading instead of concat.

Reply:
I suggest reading the comment right above that code.